Python2 incompatible syntax in views.py

scholia/app/views.py:2151:56: E999 SyntaxError: invalid syntax introduced in https://github.com/WDscholia/scholia/pull/1513/commits/114860dece24b54c5abe5d45bcdfd7386ec073b7

--- a/scholia/app/views.py
+++ b/scholia/app/views.py
@@ -2148,6 +2148,7 @@ def show_aspect_missing(aspect, q):
 
     """
     try:
-        return render_template(f'{aspect}_curation.html', q=q)
+        return render_template('{aspect}_curation.html'.format(aspect=aspect),
+                               q=q)
     except TemplateNotFound:
         return render_template("404.html")

